.travis.yml

```diff
@@ -7,8 +7,8 @@ services:
   - docker
 
 go:
-  - 1.6
-  - 1.7
+  - '1.11.x'
+  - '1.x'
```

Comment:
Do we care about these? I would just do 1.11

Reply:
Last 2 releases are supported Go. If we want only one version, we probably want 1.12.x then.

Reply:
Oh right, 1.12 is already out. Let's do 1.11 and 1.12 then